python/lib/dependabot/python/update_checker/index_finder.rb

```diff
@@ -120,6 +120,8 @@ def pyproject_index_urls
           sources.each do |source|
             if source["default"]
               urls[:main] = source["url"]
+            elsif source["name"] == "PyPI"
+              urls[:main] = "https://pypi.org/pypi/simple/"
```

Comment:
This URL is actually incorrect, it is `pypi.org/simple`:
```suggestion
              urls[:main] = "https://pypi.org/simple/"
```

Reply:
Updated this PR to correct that (with a slightly simpler approach).